spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
+require 'emerald/parser'
+require 'emerald/compiler'
+
```

Comment:
I think that example is actually semantically different as failing tests would not unlink the file unless an after block is used.

I'd prefer it all in one place though. :)